### PR TITLE
additional column

### PR DESCRIPTION
**Pull request by babakw**
please do review

### CODE REVIEW

File: create_table3 [hunk 1]
```diff
@@ -1,6 +1,7 @@
   
 create multiset table3(
 a varchar(10),
-zzzzzzzzz decimal(38,10)
+zzzzzzzzz decimal(38,10),
+sdasads decial(38,10)
```
Comment (wojtekbabak) [inline create_table3:5]: typo in data type

File: create_table3 [hunk 2]
```diff
@@ -1,6 +1,7 @@
   
 create multiset table3(
```
Comment (wojtekbabak) [inline create_table3:2]: fsdfsdfsd

**Review by wojtekbabak — CHANGES_REQUESTED** [review]
please fix this

File: create_table3 (hunk 1 above)
Comment (babakw) [reply create_table3:5]: fixed

File: create_table3 (hunk 2 above)
Comment (babakw) [reply create_table3:2]: ok done
```suggestion
create multiset table3(fgdf
```

**Review by wojtekbabak — COMMENTED** [review]
everything looks fine

**Review by wojtekbabak — APPROVED** [review]

**Comment by babakw** [comment]
test

**Pull request closed without merging**